config_defaults/defaults.ini

```diff
@@ -53,7 +53,8 @@ remove_after_test = yes
 #: CSV of possibly existing **full** image names to preserve.
 preserve_fqins = docker.io/stackbrew/centos:7,
                  docker.io/stackbrew/centos:centos7,
-                 docker.io/stackbrew/centos:latest
+                 docker.io/stackbrew/centos:latest,
+                 p4321:latest
```

Comment:
I don't think you really want this here. What this does is, when tests finish, garbage collection leaves `p4321:latest` lying around. This is pretty bad, because if the subtest changes it will be running the old version. You want this to be a throwaway image, one that gets deleted after each use.